runtime/src/storage/mkvs/sync/mod.rs

```diff
@@ -5,15 +5,86 @@ mod merge;
 mod noop;
 mod proof;
 mod stats;
-mod sync;
 
 pub use errors::*;
 pub use host::*;
 pub use merge::*;
 pub use noop::*;
 pub use proof::*;
 pub use stats::*;
-pub use sync::*;
+
+use std::any::Any;
+
+use anyhow::Result;
+use io_context::Context;
+
+use crate::{
+    common::crypto::hash::Hash,
+    storage::mkvs::{tree::*, Prefix},
+};
+
+/// Identifies a specific tree and a position within that tree.
+#[derive(Clone, Debug, Default, PartialEq, cbor::Encode, cbor::Decode)]
+pub struct TreeID {
```

Comment:
same question about this, moved from sync.rs

Reply:
`clippy::module_inception` because these were in `sync::sync` module, now these are in `sync` module directly

Reply:
thanks, didn't know about that